tests/cli/clean/conftest.py

```diff
@@ -1,14 +1,88 @@
 """Fixtures for cli clean tests"""
-
+from pathlib import Path
 import pytest
+import datetime as dt
+
+from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
+from cg.apps.balsamic.api import BalsamicAPI
+
+
+@pytest.fixture
+def balsamic_clean_store(base_store, helpers):
```

Comment:
Looks really nice! I would advocate to use type hints and return types for fixtures as well as it makes the code more readable. And docstrings of course! Then I can avoid reading the code to understand what I get from a fixture

Reply:
type hints added